quickwit/quickwit-indexing/src/source/ingest/mod.rs

```diff
@@ -116,10 +117,7 @@ impl ClientId {
 enum IndexingStatus {
     #[default]
     Active,
-    // We have emitted all documents of the pipeline until EOF.
-    // Disclaimer: a complete status does not mean that all documents have been indexed.
-    // Some document might be still travelling in the pipeline, and may not have been published
-    // yet.
+    ReachedEof,
```

Comment:
It will show up after you rebase, but please add the comment in there.
We don't want any "floating semantics".